.travis.yml

```diff
@@ -114,7 +114,8 @@ before_cache:
   - docker save dash-builder-$BUILD_TARGET-$TRAVIS_JOB_NUMBER $(docker history -q dash-builder-$BUILD_TARGET-$TRAVIS_JOB_NUMBER | grep -v \<missing\>) | gzip -2 > $HOME/cache/docker/dash-builder-$BUILD_TARGET.tar.gz
 
 before_install:
-  - travis_retry travis_apt_get_update
+  - sudo rm -rf "${TRAVIS_ROOT}/var/lib/apt/lists/"*
+  - travis_retry sudo sudo apt-get update 2>&1
   - travis_retry sudo apt-get -yq --no-install-suggests --no-install-recommends install docker-ce realpath
```

Comment:
Can't we remove this entirely if all we are installing is `docker-ce` and we have the `services: docker` set up above?

(Not sure what `realpath` is used for.)

Reply:
`realpath` must be installed in Trusty as it's not available in the base install. In Bionic it became part of coreutils. I'm investigating an upgade to Bionic atm where I already had to remove the realpath package.

Reply:
Ok, thanks. That was actually a tangent of my original question tho -- we shouldn't need to do this whole apt-get update / install thing at all since we have the `docker` service setup, right? Meaning all this would be unnecessary.

Reply:
If I remember correct, I added this to make sure we're on a recent docker version. With the Bionic upgrade, that probably won't be needed as well.